### DISCUSSION TITLE
Is there intended way to add management IP/fqdn for cloud VMs?

### DISCUSSION THREAD

**Discussion by stavr666** [netbox-community/netbox#10400] [Q&A]
Clusters seems to have no ways to add IP or DNS record.
It's not (real) problem with clusters, containing devices, since there are numerous ways to add LAG or FHRP group to every node.

But when I create cluster to manage VMs on any external cloud, there is no real devices, that I know about. So, there is no (with exception of custom fields) way to add entry point for orchestration scrips.

Shouldn't this be functionality available out of the box? Nowadays almost every infrastructure use hybrid clouds this way or another.

Netbox v3.3.2

**Comment by jcollie** [comment]
Why not set the name of the cluster to whatever magic string your orchestration tool needs to access the cluster? Otherwise, I'd go with a custom field. They're incredibly useful.
